Add a dummy Makefile target for ACL automake to do nothing during build.
Otherwise, make-ing this package re-signs the database with a newly generated
private key. Prefer to trust the package author.

diff -ru wireless-regdb-2018.09.07.orig/Makefile wireless-regdb-2018.09.07/Makefile
--- wireless-regdb-2018.09.07.orig/Makefile	2018-09-08 04:25:27.000000000 +1000
+++ wireless-regdb-2018.09.07/Makefile	2018-10-23 08:27:23.552063169 +1000
@@ -45,6 +45,8 @@
 
 all: $(REGDB_CHANGED) regulatory.bin sha1sum.txt regulatory.db.p7s
 
+nop:
+
 clean:
 	@rm -f *.pyc *.gz
 
